package.json

```diff
@@ -139,6 +138,7 @@
     "pretty-bytes": "4.0.2",
     "prop-types": "15.6.0",
     "react-addons-shallow-compare": "15.6.2",
+    "react-dates": "15.5.3",
```

Comment:
Will we need to keep updating this as their version numbers go up? What's the reasoning behind which version number we choose here? The latest one is in the 16s right now.

Reply:
The different versions have somewhat different behaviors. I noticed a bug in v16 that isn't in v15 that broke day templates, so I will update the version when that is fixed.